themes/nord.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Yavko",
```

Comment:
```suggestion
        "name": "Arctic Ice Studio & Sven Greb",
```

Attribution should go to the original theme creator. In this case, it's Arctic Ice Studio & Sven Greb.

Reply:
oops didnt know